include/revng/Pipeline/Invokable.h

```diff
@@ -367,16 +389,16 @@ void invokePipeFunction(Context &Ctx,
   constexpr auto
     ExtraArgsIndexes = std::make_integer_sequence<size_t, ExtraArgsCount>();
 
-  detail::invokeImpl<ContextType>(Ctx,
-                                  Pipe,
-                                  F,
-                                  Containers,
-                                  ArgsNames,
-                                  static_cast<ContainersTypes *>(nullptr),
-                                  static_cast<ExtraArgsTypes *>(nullptr),
-                                  Indexes,
-                                  ExtraArgs,
-                                  ExtraArgsIndexes);
+  return detail::invokeImpl<ContextT>(Ctx,
+                                      Pipe,
+                                      F,
+                                      Containers,
+                                      ArgsNames,
+                                      static_cast<ContainersTypes *>(nullptr),
+                                      static_cast<ExtraArgsTypes *>(nullptr),
+                                      Indexes,
+                                      ExtraArgs,
+                                      ExtraArgsIndexes);
```

Comment:
Invokable is getting more and more complex. Can you please add doxygen to it detailing all of its features?